docs/codeql/writing-codeql-queries/about-codeql-queries.rst

```diff
@@ -58,6 +58,7 @@ Query metadata is used to identify your custom queries when they are added to th
     Queries that are contributed to the open source repository, added to a query pack in LGTM, or used to analyze a database with the :ref:`CodeQL CLI <codeql-cli>` must have a query type (``@kind``) specified. The ``@kind`` property indicates how to interpret and display the results of the query analysis:
 
     - Alert query metadata must contain ``@kind problem``.
+    - Diagnostic query metadata must contain ``@kind diagnostic``.
     - Path query metadata must contain ``@kind path-problem``.
```

Comment:
Would be slightly nicer to put "diagnostic" either first or last so it doesn't some _between_ the alert kinds.

Reply:
Good point - I added it alphabetically, but am happy to change the order.